internal/compiler/fuzz/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "i-slint-compiler-fuzz"
+version = "0.0.0"
+authors = ["Automatically generated"]
```

Comment:
By what?
But the test will complain that the license is missing and stuff

Reply:
I admit I have not looked into this file too much: `cargo fuzz init` created it!